PokeCharts.IntegrationTests/Client/PokeAPIClientTests.cs

```diff
@@ -0,0 +1,106 @@
+using NUnit.Framework;
+using Newtonsoft.Json.Linq;
+using System.Linq;
+
+namespace Client.nUnitTests
```

Comment:
Namespace does not correspond to file location, must be: 'PokeCharts.IntegrationTests.Client'